Feature capstone

darikmohammed opened the pull request:
# World Football Global Summit 
<img width="1440" alt="Screen Shot 2022-05-18 at 11 12 59 AM" src="https://user-images.githubusercontent.com/56404835/168990970-34329192-9c15-4915-8eef-43ad21bbf599.png">
<img width="400" alt="Screen Shot 2022-05-18 at 11 13 31 AM" src="https://user-images.githubusercontent.com/56404835/168991067-e2c9b0ed-8395-4432-ba84-306de1f89fb9.png">

## Features
- Design made using the provided guidelines. 
- Personalize project 
- Use HTML5 , CSS3 and JavaScript
## Project Documentation
### Loom Recorded video 
```sh
https://www.loom.com/share/6decc6b4171d4225900097bfcda33f7c
```
### Site Deployed

```sh
https://darikmohammed.github.io/world-football-summit/
```

darikmohammed committed:
Initalize project with linter

darikmohammed committed:
Add heading and Main program page

- Create the heading mobile view
- Create the main program section for mobile view

darikmohammed committed:
Add featured-speaker and sponser section

- Created the featured speaker
- Created the sponser section  and finish up main

darikmohammed committed:
Add Footer and finish up home page mobile view

darikmohammed committed:
Add About page for mobile view

darikmohammed committed:
Update Readme with the live demo link

darikmohammed committed:
Fix about project section

darikmohammed committed:
Finish Desktop View for About page

darikmohammed committed:
Add the style for mobile menu

darikmohammed committed:
Add JavaScript for mobile menu

darikmohammed committed:
Add Javascript for adding featured speakers dynamically

darikmohammed committed:
Add 404 page not found error

darikmohammed committed:
Update link ref

darikmohammed committed:
Update link ref for WF campaign

darikmohammed committed:
Update 404 page link ref

darikmohammed committed:
Update Speaker images

darikmohammed committed:
Update Readme file

darikmohammed committed:
Update Readme with loom video

darikmohammed committed:
Fix Linter errors

darikmohammed committed:
Update README.md

Kingstalux commented on index.html:
```diff
@@ -4,11 +4,172 @@
+          </div>
+        </div>
+        <div class="join-us">
+          <h3>Join WF Global Summit 2020</h3>
```
- [x] Great job so far 👍 Kindly style this button in the desktop version as in the original design 👍 
- Your design:
![Annotation 2022-05-18 094625](https://user-images.githubusercontent.com/85113567/168998472-b727b9cb-bca4-49b5-be6b-efdfd6d35836.jpg)
- Required design:
![{EF23F69B-574B-4A23-AB0C-BD04861CA01E} png](https://user-images.githubusercontent.com/85113567/168998556-5bb2663d-af9b-48c6-8ddc-c416249fe386.jpg)

Kingstalux commented on about.html:
```diff
@@ -0,0 +1,139 @@
+        <p>2020 World Football Club</p>
+        <p>Some Rights Reserved</p>
+      </div>
+    </footer>
```
- [x] The about page does not have a partner's section in the desktop view. It instead has a dark footer. Kindly address this issue so as to follow the original design 😄 
- Your design:
![Annotation 2022-05-18 093112](https://user-images.githubusercontent.com/85113567/168999253-350ccf38-707f-4f31-ad03-ab529d8745fd.jpg)
- Required design:
![Annotation 2022-05-18 093043](https://user-images.githubusercontent.com/85113567/168999307-c22ef82b-37f6-42cb-bb52-95f379cfd457.jpg)

Kingstalux commented on Assets/script/index.js:
```diff
@@ -21,3 +59,20 @@ closeMenuBtn.addEventListener("click", () => {
+      </p>
+    </div>
+    `;
+}
```
- [x] Great job creating the speaker's section dynamically from JavaScript 💯 In the mobile view, only 2 speakers should be displayed together with the `more` button, and when a user clicks the `more` button the rest of the speakers are then displayed. Kindly add this feature 👍 
- Your design:
![{26DB3112-DFD4-4F2A-BE5A-6B82B60FC626} png](https://user-images.githubusercontent.com/85113567/169000061-a517fe37-a507-4a4a-8d2b-d0547a339bc9.jpg)
- Required design:
![Annotation 2022-05-18 092722](https://user-images.githubusercontent.com/85113567/169000101-05d7b3ec-2e9f-45fc-9319-d9d410caf2bd.jpg)

darikmohammed replied:
@Kingstalux Kindly this more button is optional one it is stated in the requirement page.
<img width="1217" alt="Screen Shot 2022-05-18 at 12 49 33 PM" src="https://user-images.githubusercontent.com/56404835/169011255-fd524f19-8244-467d-823c-b6959e6c95d2.png">

Kingstalux replied:
Hello @darikmohammed, since the functionality of the `more` button is optional, just stick to the original design and display 2 speakers in the mobile view with the button but don't add the functionality 👍

darikmohammed replied:
@Kingstalux Thank you.

Kingstalux reviewed (changes requested):
Hi @darikmohammed, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

![68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f744b797263304e546c4563475764546870342f67697068792e676966](https://user-images.githubusercontent.com/85113567/169000574-b3b9323f-0356-4481-b124-5f09dcf366a0.gif)

## Highlights 👏 
✅ Good responsive design 👍 
✅ Professional README file 💯 
✅ Good commit history 👏 
✅ Good PR description 🥂 


## Suggested changes
_Check the comments under the review._

- [ ] In your presentation you are supposed to present all features of the project both in the mobile and the desktop view. You are also supposed to talk about a piece of code you found interesting or you are proud of while making this project. Kindly redo the video presentation and include these additional requirements 💯 

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

darikmohammed committed:
Update for request changes

- Style Join-us button in the desktop version as in the original design
- Update the about page footer and partner's section in desktop view
- Update on More and Less button in mobile view of home page. Implemented dynamically.

darikmohammed committed:
Fix Linter errors

darikmohammed committed:
Update the mobile menu

darikmohammed commented:
Updated the Loom video in the PR

gemmen29 reviewed (approved):
Hi @darikmohammed, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

![](https://media.giphy.com/media/mCIjCgs3nWQWfJZvPA/giphy-downsized-large.gif)

## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
